src/cider_nrepl/plugin.clj

```diff
@@ -27,15 +27,15 @@
                                   ;; In this case the onus is on the user. A warning will be emitted
                                   ;; later, but we assume that the user will provide an appropriate
                                   ;; implementation.
-                                  ,,true
+                                  true
 
```

Comment:
This usage of `,,true` seems intentional... are the commas a convention of some sort that cljfmt doesn't know about?

Reply:
No, they're just an @arrdem idiosyncrasy.

Reply:
weavejester/cljfmt#67 it me

Reply:
Ok to remove them and make cljfmt happy? Otherwise I can leave them, and you'll have warnings in CI until/unless weavejester/cljfmt#67 gets addressed.  My vote would be remove the commas to avoid masking other formatting issues from creeping in.

Reply:
Yes, remove them. 🔨  @arrdem's dreams of being a stylistic trendsetter.

Reply:
Ok, then that's what merging this will do.